source/extensions/filters/network/postgres_proxy/postgres_decoder.h

```diff
@@ -91,7 +89,8 @@ class DecoderImpl : public Decoder, Logger::Loggable<Logger::Id::filter> {
   // first - string with message description
   // second - vector of Decoder's methods which are invoked when the message
   // is processed.
-  using MsgProcessor = std::pair<std::string, std::vector<MsgAction>>;
+  using MsgProcessor =
```

Comment:
I never liked too much of the abbreviated version of `Message` word. What about rename all `Msg*` to `Message*`?

Reply:
I can change that for consistency and use `Message`.